src/Utilities.js

```diff
@@ -0,0 +1,24 @@
+export default class Utilities {
+    static add = (data, task) => {
+      task.index = data.length;
+      task.completed = false;
+      data.push(task);
+      return data;
+    }
```

Comment:
- [ ] Tasks added to the todo list should have 1-based indexes, which means that the property `index` should start from 1. 
(i.e. if a 5th task is added to the list, the index of that task should equal to 5). Currently, it starts from 0, please go ahead and fix that.

![Screenshot from 2021-09-30 19-21-06](https://user-images.githubusercontent.com/40503704/135510771-0b541836-18bf-4591-b910-20d3db656874.png)

Reply:
This issue is well resolved! ✔️